src/common/CalculateRotationZ.inc.c

```diff
@@ -1,7 +1,19 @@
 #include "common.h"
```

Comment:
I think you should call this file CalculateArcsinDeg and rename the function CalculateRotationZ -> CalculateArcsinDeg accordingly.